### PR TITLE
Add --force option

### LINKED ISSUE #14: Feature Request: Add --force command to eav:media:remove-unused
I added eav:media:remove-unused command to a cron due to how my Magento site adds products, it syncs up images nightly and wipes all product images with new ones. Magento leaves the old images on the hard disk so I want it to be cleaned up daily.

I added eav:media:remove-unused to a cron but because of the prompt I don't think it runs - I amended the vendor file to do this which works for me but am scared of it being overwritten in the future - hence the feature request! Otherwise I'll make a plugin to properly extend it.

```
protected function configure()
    {
        $this
            ->setName('eav:media:remove-unused')
            ->setDescription('Remove unused product images')
            ->addOption('dry-run')
			->addOption('force');
    }

```
 ```
   public function execute(InputInterface $input, OutputInterface $output): void
    {
        $fileSize   = 0;
        $countFiles = 0;
        $isDryRun   = $input->getOption('dry-run');
        $isForce   = $input->getOption('force');

        if (!$isDryRun && $input->isInteractive() && !$isForce) {
```

Adding a --force command would be useful for those who want this in a cron.

### PR DESCRIPTION
**Pull request by fredden**
- Adds a new option `--force` to bypass the prompt - this fixes #14
- Fixes the return type for all commands so they correctly report success/failure.

### CODE REVIEW

File: Console/Command/CleanUpAttributesAndValuesWithoutParentCommand.php [hunk 1]
```diff
@@ -42,19 +42,27 @@ protected function configure()
         $this
             ->setName('eav:clean:attributes-and-values-without-parent')
             ->setDescription($description)
-            ->addOption('dry-run');
+            ->addOption('dry-run')
+            ->addOption('force');
     }
 
     public function execute(InputInterface $input, OutputInterface $output)
```
Comment (sprankhub) [inline Console/Command/CleanUpAttributesAndValuesWithoutParentCommand.php]: We could add `int` as a return type now.

File: Console/Command/RemoveUnusedAttributesCommand.php [hunk 2]
```diff
@@ -45,17 +45,27 @@ protected function configure()
         $this
             ->setName('eav:attributes:remove-unused')
             ->setDescription('Remove unused attributes (without values or not assigned to any attribute set')
-            ->addOption('dry-run');
+            ->addOption('dry-run')
+            ->addOption('force');
     }
 
     public function execute(InputInterface $input, OutputInterface $output)
```
Comment (sprankhub) [inline Console/Command/RemoveUnusedAttributesCommand.php]: We could add `int` as a return type now.

**Review by sprankhub — APPROVED** [review]

File: Console/Command/CleanUpAttributesAndValuesWithoutParentCommand.php (hunk 1 above)
Comment (fredden) [reply Console/Command/CleanUpAttributesAndValuesWithoutParentCommand.php]: Thanks for catching this. I've updated this now as intended initially.

File: Console/Command/RemoveUnusedAttributesCommand.php (hunk 2 above)
Comment (fredden) [reply Console/Command/RemoveUnusedAttributesCommand.php]: Thanks for catching this. I've updated this now as intended initially.

**Comment by sprankhub** [comment]
Thank you for your contribution!

**Pull request merged**